Adyen/Utilities/Environment.swift

```diff
@@ -12,6 +12,12 @@ public struct Environment: APIEnvironment {
     /// :nodoc:
     public var baseURL: URL
     
+    /// :nodoc:
+    public var headers: [String: String] = ["Origin": "https://test.org", "Content-Type": "application/json"]
+    
+    /// :nodoc:
+    public var queryParameters: [URLQueryItem] = [URLQueryItem(name: "token", value: "test_YP5RJK3TL5EELNPAS7RHIRVRAIMYZXOH")]
```

Comment:
The `token` value is generated for a specific `origin in the backoffice, so it is different for different Accounts (and different `origin`).
It should be a parameter, I believe..

Reply:
the client key injection will be the subject my next pull request :-) , I don't want to make huge PR again :-D .